task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -57,24 +63,23 @@ spec:
         check_num=4
         failure_num=0
         TESTPASSED=true
-        chmod +x opm grpc_health_probe
 
-        if ! ./opm version; then
-          echo "!FAILURE! - opm binary check failed"
+        if [ ! $(find . -name "opm") ]; then
+          echo "!FAILURE! - opm binary presence check failed"
           failure_num=`expr $failure_num + 1`
           TESTPASSED=false
         fi
-        if [ ! -f "grpc_health_probe" ]; then
-          echo "!FAILURE! - grpc_health_probe binary check failed"
+        if [ ! $(find . -name grpc_health_probe ) ]; then
+          echo "!FAILURE! - grpc_health_probe binary presence check failed"
           failure_num=`expr $failure_num + 1`
           TESTPASSED=false
         fi
-        if ! ./opm validate confdir; then
+        if ! opm validate ."${conffolder}"; then
```

Comment:
Did we validate that conffolder is non-empty to avoid weird errors?

Reply:
Check added.